runtime-config.md

```diff
@@ -12,30 +12,7 @@ Only [mounts from the portable config](config.md#mount-points) will be mounted.
 
 *Example (Linux)*
 
-```json
-"mounts": {
-    "proc": {
-        "type": "proc",
-        "source": "proc",
-        "options": []
-    },
-    "dev": {
-        "type": "tmpfs",
-        "source": "tmpfs",
-        "options": ["nosuid","strictatime","mode=755","size=65536k"]
-    },
-    "devpts": {
-        "type": "devpts",
-        "source": "devpts",
-        "options": ["nosuid","noexec","newinstance","ptmxmode=0666","mode=0620","gid=5"]
-    },
-    "data": {
-        "type": "bind",
-        "source": "/volumes/testing",
-        "options": ["rbind","rw"]
-    }
-}
-```
+See Mounts [example](runtime-config-linux.md#mounts-in-linux) in Linux
 
 *Example (Windows)*
```

Comment:
On Fri, Sep 11, 2015 at 09:25:56AM -0700, Daniel, Dao Quang Minh wrote:
> >  *Example (Windows)*
> 
> I think this makes sense. but probably a different PR.

Fair enough.  I'm fine with this PR landing as it stands with 5ab8432
and filing subsequent PRs to clean things up more (but that approach
has been slow before, e.g. see #141 ;).  Of my outstanding comments,
the overspecified header seems the lowest-hanging fruit [1], although
[2,3] seem easy and uncontroversial as well.

[1]: https://github.com/opencontainers/specs/pull/171/files#r39280224
[2]: https://github.com/opencontainers/specs/pull/171/files#r39280300
[3]: https://github.com/opencontainers/specs/pull/171/files#r39280066